plugin/src/test/groovy/com/novoda/buildproperties/test/ExtendedTruth.groovy

```diff
@@ -0,0 +1,144 @@
+package com.novoda.buildproperties.test
+
+import com.google.common.annotations.GwtIncompatible
+import com.google.common.base.Optional
+import com.google.common.collect.*
+import com.google.common.truth.*
+import com.google.common.util.concurrent.AtomicLongMap
+import com.novoda.buildproperties.CompositeException
+import com.novoda.buildproperties.Entry
+
+import javax.annotation.Nullable
+
+/**
+ * This utility class serves as static extension of {@link Truth},
```

Comment:
Here is the link :) http://mrhaki.blogspot.de/2013/01/groovy-goodness-adding-extra-methods.html